### ISSUE TITLE
Unable to start a second RTMP stream on iOS

### ISSUE THREAD

**Issue by izquiratops** [ezranbayantemur/react-native-rtmp-publisher#6]
The very first time works fine, but stopping the stream and starting a new one will crash. This is happening only on iOS.

**To Reproduce**
1. Run .startStream()
2. Stream works
3. Run .stopStream()
4. Run .startStream() again
5. Stream won't start. RTMP client is not able to proceed the connection after the handshake.

**Expected behavior**
Should start a new stream every time I run startStream. 

**Smartphone:**
 - Device: iPhone 13 and iPhone 11
 - OS: both with iOS 16.1
 - react-native-rtmp-publisher: 0.4.4

**Additional context**
I tried [apiVideo](https://github.com/apivideo/api.video-reactnative-live-stream), which uses HaishinKit 1.3.0 and works as expected, maybe this is an issue comes from 1.2.7.

I tried to fork it and upgrade it but I've never tried Xcode before and I'm kinda lost

**Comment by ezranbayantemur** [comment]
I'll be looking at it soon 👍🏻

**Comment by izquiratops** [comment]
Maybe this thing happens because stream.attachCamera(nil)/stream.attachAudio(nil) is never called, removeFromSuperview could be a great place to write those lines. I tried to build it a couple of times but XCode is always throwing the build with a lot of errors from HaishinKit

**Comment by ezranbayantemur** [comment]
@izquiratops I'm currently updating the HaishinKit to 1.4.1 but there is a problem about the process. I've created an issue about it [on here](https://github.com/shogo4405/HaishinKit.swift/issues/1119)

I'm working about it

**Comment by turkmenkaan** [comment]
I was wondering if there are any updates on this.

**Comment by izquiratops** [comment]
Yes, I found out that this bug is fixed if I create a new RTMPStream object every time I run startPublish like this:
```swift
public static func startPublish(){
        connection.requireNetworkFramework = true
        connection.connect(_streamUrl)
        self.stream = RTMPStream(connection: connection)
        self.stream.publish(_streamName)
        isStreaming = true
}
```

Since this seems a pretty crappy solution I hope @ezranbayantemur may do something about it 😅

**Comment by Andriiklymiuk** [comment]
Used suggested @izquiratops  changes to fix that.
Also updated haishinkit to 1.4.3

https://github.com/ezranbayantemur/react-native-rtmp-publisher/pull/38
